### ISSUE TITLE
Support for HWE install

### ISSUE THREAD

**Issue by h4ndshake** [anbox/anbox#26]
I guess the title says it all, Canonical recently launched it's HWE Stack, wich updates Xorg, Kernel, and that kind of stuff on xenial. Currently anbox installer does not support systems with HWE, installing linux-headers-generic (4.4) instead of linux-headers-generic-hwe-16.04.

**Comment by morphis** [comment]
Good point, are you interested in fixing this as part of the anbox-installer? See https://github.com/anbox/anbox-installer/blob/master/installer.sh#L121 for details. The installation step is a bit lazy and I think with a bit parsing of which is the currently used kernel package we can make this happen.

**Comment by h4ndshake** [comment]
Sorry for the late response.
It seems that on newer ubuntu installations (16.04.2) the HWE stack is provided by default, but i'm not sure about Installations that were upgraded to 16.04.2 (ex: 16.04 > 16.04.2)
Distros based on older ubuntu versions (ex: elementary OS have a ISO based on 16.04) doesn't come with HWE.

The question is, we have the intention for supporting other distros?

**Comment by h4ndshake** [comment]
And the depencencie issue is in the anbox-modules-dkms package, not in the installer script.
